Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -210,7 +210,7 @@ extension TopologyDescription.TopologyType {
 }
 
 public struct TestError: LocalizedError {
```

Comment:
Errors thrown from within the tests don't need to be the same ones the driver throws, so in a lot of places I converted throwing driver errors to throwing this one to preserve non-`@testable` importing.